### ISSUE TITLE
Error while scanning

### ISSUE THREAD

**Issue by Sailorrabbit** [arsenetar/dupeguru#280]
What does this mean?

```
Traceback (most recent call last):
  File "C:\src\dupeguru\hscommon\gui\progress_window.py", line 92, in pulse
  File "C:\src\dupeguru\hscommon\jobprogress\performer.py", line 59, in reraise_if_error
  File "C:\src\dupeguru\hscommon\jobprogress\performer.py", line 43, in _async_run
  File "C:\src\dupeguru\core\app.py", line 724, in do
  File "C:\src\dupeguru\core\scanner.py", line 128, in get_dupe_groups
  File "C:\src\dupeguru\core_pe\scanner.py", line 21, in _getmatches
  File "C:\src\dupeguru\core_pe\matchblock.py", line 157, in getmatches
  File "C:\src\dupeguru\core_pe\matchblock.py", line 76, in prepare_pictures
  File "C:\src\dupeguru\core_pe\photo.py", line 104, in get_blocks
  File "C:\src\dupeguru\qt\pe\app.py", line 46, in _plat_get_blocks
TypeError: unorderable types: int() <= str()
```

**Comment by hsoft** [comment]
Hard to tell just from the traceback. The culprit line is https://github.com/hsoft/dupeguru/blob/master/qt/pe/app.py#L46 , so it seems that a bad `orientation` (a value coming from a JPEG) is the cause of it.

Our best bet would be for you to re-run the scan in Debug mode. It will output the name of the file being processed before-hand so we'll be able to know what picture is responsible for the crash. Then, you can upload it here so we can reproduce the problem.

**Comment by arsenetar** [comment]
Fixed by #698
